SIG.md

```diff
@@ -0,0 +1,107 @@
+
+# SIG Roles
+## SIG Lead
+
+- Membership tracked in sigs.yaml
+- MUST be an odd number of SIG Leads in a group.
+- MUST be a reviewer of one or more areas of the project, and should be a subject matter expert in their SIG area.
+- SHOULD define how priorities and commitments are managed. MAY delegate to other leads as needed.
+- SHOULD drive charter changes (including creation) to get community buy-in but MAY delegate content creation to SIG contributors.
+- MUST identify, track, and maintain the metadata of the SIGs enhancements for the current release and serve as point of contact for the release team, but MAY delegate to other contributors to fulfill these responsibilities.
+- MUST organize a main group meeting and make sure sigs.yaml is up to date.
+- SHOULD facilitate meetings but MAY delegate to other members or future chairs/chairs in training.
+- MUST ensure meetings are recorded and at least notes made available.
+- MUST coordinate communication and be a connector with other community groups like SIGs and the committers but MAY delegate the actual communication and creation of content to other contributors where appropriate.
+
+### Requirements
+Chairs MUST be at least a "member" on our [contributor ladder](https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md#organization-member) to be eligible to hold a leadership role within a SIG.
+SIGs MAY prefer various levels of domain knowledge depending on the role. This should be documented.
+
+
+### On-boarding and Off-boarding Leads
+Chairs MAY decide to step down at anytime and propose a replacement. Use lazy consensus amongst the other Chairs with fallback on majority vote to accept the proposal. The candidate SHOULD be supported by a majority of SIG contributors.
+Chairs MAY select additional leads through a super-majority vote amongst Chairs. This SHOULD be supported by a majority of SIG contributors.
+
+## Other Roles
+This governance document outlines the required roles for SIGs; however, SIGs are allowed to operate how they see fit outside of minimum governance requirements, including defining more roles to sustain the group. If a SIG needs to change the Chair position to include or remove duties, this needs to be approved by the committers. Newly created roles that don't assume any responsibility of Chairs should follow the governing processes in the SIG's charter.
+
+# SIG Creation
+
+## Goals
+The primary goal of the charters is to define the scope of the SIG within Cilium and how the SIG leaders exercise ownership of these areas by taking care of their responsibilities. A majority of the effort should be spent on these concerns.
+
+### SIG Charter Guide
```

Comment:
We talked about having SIG Leads taking a particular role in reviewing CFPs. I think we should spell that out either here in this document, or in the Charter Guide. 

The intent is offload work from the committers, so they aren't a bottleneck on CFP reviews
 - expectation that SIG leads would be responsible for reviewing CFPs in their area
 - they have a responsibility to bring concerns to committers if they are unsure

I think also we might need to spell out that SIGs don't have additional authority - they are expected to drive towards solutions that can be accepted by the project's committer community (ideally by consensus, but if necessary by a vote)

Reply:
good point! added in the "SIGs don't have additional authority" near the top of the file